src/include/OSL/oslnoise.h

```diff
@@ -145,24 +145,13 @@ namespace {
 
 // return the greatest integer <= x
 inline OSL_HOSTDEVICE int quick_floor (float x) {
-    return (int) x - ((x < 0) ? 1 : 0);
+    return (int)floorf(x);
 }
 
 #ifndef __CUDA_ARCH__
 // return the greatest integer <= x, for 4 values at once
 OIIO_FORCEINLINE int4 quick_floor (const float4& x) {
```

Comment:
Same here ... why not just call `ifloor` directly everywhere?

Reply:
OK, updated the PR with this rename as requested. The reason I didn't include it at first was that it swamps and obscures the actual bug fix, making the review confusing. But if you like the rename, I'll merge it together.